evaluation.md

```diff
@@ -5,6 +5,33 @@
 _Describe each change from your ideal syntax to the syntax you implemented, and
 describe_ why _you made the change._
 
+The first compromise we had to make was changing from "your nth weapon" to "weapon 1" because we didn't want to
+implement 99 different functions to represent the different ordinal numbers.
+
+The second compromise we had to make was making sure every rule had an odd number of words - this is because for the programmer to be able to write their code without dots and parentheses, every method needed to take 1 argument. Including the `If` in the front, this meant that every rule had to have odd length.
```

Comment:
Here's some stuff that may come in handy later.
http://stackoverflow.com/questions/1181533/what-are-the-precise-rules-for-when-you-can-omit-parenthesis-dots-braces-f